packages/react/src/components/Storage/StorageManager/StorageManager/types.ts

```diff
@@ -47,20 +47,24 @@ export interface StorageManagerProps {
    * Maximum file size in bytes
    */
   maxFileSize?: number;
+  /**
+   * When a file is removed
+   */
+  onFileRemove?: (file: { key: string }) => void;
   /**
    * Monitor upload errors
    */
   onUploadError?: (error: string) => void;
   /**
-   * Monitor upload sucess
+   * Monitor upload success
    */
   onUploadSuccess?: (event: { key: string }) => void;
   /**
    * Process file before upload
    */
   processFile?: (
-    storageFile: Required<Pick<StorageFile, 'file' | 'name'>>
-  ) => Required<Pick<StorageFile, 'file' | 'name'>>;
+    storageFile: Required<Pick<StorageFile, 'file' | 'key'>>
+  ) => Required<Pick<StorageFile, 'file' | 'key'>>;
```

Comment:
For legibility, what do you think of making a utility type from `Required<Pick<StorageFile, 'file' | 'key'>>`, example:
```
type ProcessFileParams = Required<Pick<StorageFile, 'file' | 'key'>>;

...

processFile?: (params: ProcessFileParams) => ProcessFileParams;
```